survey-skill-template/lambda/custom/index.js

```diff
@@ -73,6 +74,42 @@ const LaunchRequestHandler = {
   },
 };
 
+const StartMyStandupIntentHandler = {
+  canHandle(handlerInput) {
+    return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+      && handlerInput.requestEnvelope.request.intent.name === 'StartMyStandupIntent';
+  },
+  async handle(handlerInput) {
+    const requestAttributes = handlerInput.attributesManager.getRequestAttributes();
+    let speakOutput;
+    const repromptOutput = requestAttributes.t('GREETING_REPROMPT');
+    const person = personalization.getPerson(handlerInput);
+    if (person) {
+      speakOutput = requestAttributes.t('PERSONALIZED_GREETING', personalization.getPersonalizedPrompt(handlerInput));
+      const upsServiceClient = handlerInput.serviceClientFactory.getUpsServiceClient();
+      const profileEmail = await upsServiceClient.getProfileEmail();
+      const profileName = await upsServiceClient.getProfileName();
+      const sessionAttributes = handlerInput.attributesManager.getSessionAttributes();
+      sessionAttributes.userEmail = profileEmail;
+      sessionAttributes.userName = profileName;
+      return handlerInput.responseBuilder
+        .addDelegateDirective({
+          name: 'GetReportIntent',
+          confirmationStatus: 'NONE',
+          slots: {},
+        })
+        .speak(speakOutput)
```

Comment:
Can we add reprompt here ? This is for handling cases where we dont get any response from user.

https://developer.amazon.com/en-US/docs/alexa/custom-skills/manage-skill-session-and-session-attributes.html#session-lifecycle